.github/workflows/actions-test.yml

```diff
@@ -89,7 +97,8 @@ jobs:
           prefix: "Prefix-"
           suffix: "-Suffix"
 
-      - name: Created workspace is not symlkink
+      - name: Created workspace is not symlkin
```

Comment:
This still seems to be typo. Can you please fix all `symlkin` in this file to `symlink`?
These were my mistake 😲

Reply:
Thank you, fixed.